script.js

```diff
@@ -1,4 +1,89 @@
 var main = function (input) {
-  var myOutputValue = 'hello world';
+  if (
+    input == "scissors" ||
+    input == "stone" ||
+    input == "paper" ||
+    input == "Scissors" ||
+    input == "Stone" ||
+    input == "Paper"
+  ) {
+    var myOutputValue = playGame(input);
+    return myOutputValue;
+  } else if (input == null) {
+    alert("Please enter a valid value");
+  } else {
+    alert("Please enter a valid value");
```

Comment:
You can leave out line 12 and 13. The else statement in line 14 will already cover the null input by default.